src/problems/ode_problems.jl

```diff
@@ -174,6 +174,17 @@ function Base.setproperty!(prob::ODEProblem, s::Symbol, v, order::Symbol)
     Base.setfield!(prob, s, v, order)
 end
 
+function ConstructionBase.constructorof(::Type{P}) where {P <: ODEProblem}
```

Comment:
why is this needed?

Reply:
For https://github.com/SciML/DiffEqBase.jl/pull/1070, but it might not be now.